test/utils.ts

```diff
@@ -26,12 +26,14 @@ import path from 'path';
 import {fileURLToPath} from 'url';
 
 import chai from 'chai';
+import fs from 'fs-extra';
 
 import {CompilationEnvironment} from '../lib/compilation-env';
 import {CompilationQueue} from '../lib/compilation-queue';
 import {CompilerProps, fakeProps} from '../lib/properties';
 
-export function makeCompilationEnvironment(options) {
+// TODO: Find proper type for options
+export function makeCompilationEnvironment(options: Record<string, any>): CompilationEnvironment {
```

Comment:
This might be https://github.com/compiler-explorer/compiler-explorer/blob/main/lib/options-handler.ts#L58 but we can clean up the types here in a future PR

Reply:
Doesn't seem like it since we're asking .props, .concurrency, and .queue, all of which don't really make sense on the ce options